test/unit/behaviours/tenancy-start-west-midlands-check.test.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const reqres = require('reqres');
+const Behaviour = require('../../../apps/right-to-rent-check/behaviours/tenancy-start-west-midlands-check');
+
+describe('apps/behaviours/tenancy-start-west-midlands-check', () => {
+  it('exports a function', () => {
+    expect(Behaviour).to.be.a('function');
+  });
+
+  class Base {
+      getNextStep() {}
+      saveValues() {}
+    }
+    let req;
+    let res;
+    let sessionModel;
+    let instance;
+    let ValidTenants;
+
+  beforeEach(() => {
+    sessionModel = {
+      get: sinon.stub(),
+      set: sinon.stub(),
+      unset: sinon.stub()
+    };
+    res = reqres.res();
+    req = reqres.req({sessionModel});
+    sinon.stub(Base.prototype, 'getNextStep');
+    sinon.stub(Base.prototype, 'saveValues');
+    ValidTenants = Behaviour(Base);
+    instance = new ValidTenants();
+  });
+
+  afterEach(() => {
+    Base.prototype.getNextStep.restore();
+    Base.prototype.saveValues.restore();
+  });
+
```

Comment:
IMO It's better to wrap the `beforeEach` in the `describe` for the method. Like...
```js
describe('saveValues', () => {
    beforeEach(() => {
        sinon.stub(Base.prototype, 'saveValues');
    });
    afterEach(() => {
        Base.prototype.saveValues.restore();
    });
```
This is because the `beforeEach` is run before each `it` and the `afterEach` is run after every `it` and I think nesting your tests like so ^ makes it clearer as to what is happening, i.e. what set up is for what test.